README.md

```diff
@@ -12,6 +12,7 @@ This project hosts the Java client library for the Google Ads API.
 
   * Java 1.8+
   * Maven 3.0+
+  * Protobuf 3.12.2+
```

Comment:
How frequently do you anticipate this version will need to change? Wondering if it makes sense to automate keeping this in sync with poms, like you do with the library version on line 22.

Reply:
Good point, I'll add this to the release tasks for the next release.